incent-config/Dangerfile

```diff
@@ -136,6 +136,14 @@ def strip_doc(doc)
   doc.strip.tr("\n", ' ')
 end
 
+def rendered_programs_changed?
+  `git status ./programs --short | wc -l`.strip != '0'
```

Comment:
This returns `false` if no changes are found in the `./programs` directory. Should we also add the check @jasmineHundal proposed of checking for a `./templates` folder as well or is that unneeded?

Reply:
I'm not 100% sure about this yet.

Reply:
```sh
❯ git status --short
?? edge-retailer-2021/programs/foo.yml

…/incent-config-winfield on  feature/edge-retailer-2021/add-examples [?] 
❯ git status --short ./programs

…/incent-config-winfield on  feature/edge-retailer-2021/add-examples [?] 
❯ git status --short edge-retailer-2021/programs/
?? edge-retailer-2021/programs/foo.yml
```

Reply:
I like @jasmineHundal's idea about checking for a `templates` folder, however I'm not sure the added overhead of shelling out for that check is worth it. 🤔